src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <>
+      <div className="button-panel">
```

Comment:
**[OPTIONAL]**
You could store the button name list in an array, loop over the array containing the button names and render button each time you loop over the button name array, it is much more cleaner than what we have now.

Reply:
I complete agree with the others suggestion but this dont have sense... why use logic to assign the name of ten buttons?

Reply:
@Alexoid1, Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.
Example; You wrote the `Button` component 13 times 🙂 .

You can practice DRY code by using an array and loop over it as @Forison suggested.

Reply:
> Create basic funcional componets.

You could be more specific in your PR description by stating exactly what you did.

 PR summary should be well detailed.
 
Example; 
![image](https://user-images.githubusercontent.com/31897434/103676409-562b8400-4f81-11eb-80fe-71047649bf62.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**NOTE:** We should always double-check our spellings.
`Create basic funcional componets.` instead of `Create basic functional components.`